### ISSUE TITLE
Ability hide current temperature

### ISSUE THREAD

**Issue by cannondale0815** [artem-sedykh/mini-climate-card#39]
Great addition to Home Assistant! I prefer this over all other climate cards.

Would there be a way to hide the current temperature indicator and only display the target temperature? I use a different card for displaying and tracking indoor temperature, so I would like to use your card just for adjusting target temperature. I tried the following, but got a NaN (not-a-number) value on the card:

```
          - type: 'custom:mini-climate'
            entity: climate.thermostat
            temperature:
              source: []
```

**Comment by elgeniskogen** [comment]
I also think this would be very useful :-)

**Comment by stibbzy** [comment]
I worked love to be able to hide this too!

**Comment by re-id** [comment]
Поддерживаю

**Comment by mishnz** [comment]
I've added this feature in a pull request here: https://github.com/artem-sedykh/mini-climate-card/pull/82

**Comment by regevbr** [comment]
released in 2.1.1 thanks @mishnz
